sdk/attestation/attestation/src/models/attestationResult.ts

```diff
@@ -103,7 +103,7 @@ export interface AttestationResult {
    * `runtimeJson` parameter to the `Attest` API is specified. It will
    * not be populated if the `runtimeData` parameter is specified.
    */
-  runtimeClaims: any;
+  runTimeClaims: unknown;
```

Comment:
`runTime` feels odd in comparison to `runtime`, did someone ask for this change?

Reply:
@xirzec asked me to change inittimeData to initTimeData. To be consistent in naming convention, I renamed runtimeData to match the pattern.

`inittimeData` is data which is presented at the time the enclave is initialized, `runtimeData` is data which is presented at the time that the evidence is created. The naming convention for both of these should be the same, IMHO since they are so closely related.

Reply:
Btw, `initTime` and `runTime` is also consistent with the `AttestSgxEnclaveOptions`/`AttestOpenEnclaveOptions` model types.